htdocs/commande/class/commande.class.php

```diff
@@ -879,6 +879,11 @@ public function create($user, $notrigger = 0)
 		// $date_commande is deprecated
 		$date = ($this->date_commande ? $this->date_commande : $this->date);
 
+		if (empty($this->warehouse_id) || $this->warehouse_id  = -1){
```

Comment:
$this->warehouse_id  = -1  is surely a bug. == ?

Reply:
Yes is not bug i change

Reply:
@eldy Sory an error please valid in V13